app/build.gradle

```diff
@@ -64,6 +64,8 @@ dependencies {
 
     annotationProcessor 'androidx.room:room-compiler:2.6.1'
 
+    debugImplementation 'androidx.annotation:annotation:1.7.1'
```

Comment:
This was added since without it I got the following error when running the gradle build (or the built-in Android Studio build):
```
Configuration cache state could not be cached: field `__runtimeLintModelMetadataFileCollection__` of `com.android.build.gradle.internal.lint.AndroidArtifactInput` bean found in field `__androidTestArtifact__` of `com.android.build.gradle.internal.lint.VariantInputs` bean found in field `__variantInputs__` of task `:app:generateDebugLintReportModel` of type `com.android.build.gradle.internal.lint.LintModelWriterTask`: error writing value of type 'org.gradle.api.internal.file.collections.DefaultConfigurableFileCollection'
> Could not resolve all files for configuration ':app:debugAndroidTestRuntimeClasspath'.
   > Could not resolve androidx.annotation:annotation:1.7.1.
     Required by:
         project :app
      > Cannot find a version of 'androidx.annotation:annotation' that satisfies the version constraints:
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.annotation:annotation:1.7.1'
           Constraint path 'smouldering-durtles:app:unspecified' --> 'androidx.annotation:annotation:{strictly 1.3.0}' because of the following reason: debugRuntimeClasspath uses version 1.3.0
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.test:runner:1.5.2' (runtime) --> 'androidx.annotation:annotation:1.2.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.test:core:1.5.0' (runtime) --> 'androidx.annotation:annotation:1.2.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.test.ext:junit:1.1.5' (runtime) --> 'androidx.annotation:annotation:1.2.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.test:runner:1.5.2' (runtime) --> 'androidx.test:annotation:1.0.1' (runtime) --> 'androidx.annotation:annotation:1.2.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.test:runner:1.5.2' (runtime) --> 'androidx.test:monitor:1.6.1' (runtime) --> 'androidx.annotation:annotation:1.2.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.test:runner:1.5.2' (runtime) --> 'androidx.test.services:storage:1.4.2' (runtime) --> 'androidx.annotation:annotation:1.2.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.test:runner:1.5.2' (runtime) --> 'androidx.tracing:tracing:1.0.0' (releaseRuntimePublication) --> 'androidx.annotation:annotation:1.1.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.test:core:1.5.0' (runtime) --> 'androidx.lifecycle:lifecycle-common:2.5.1' (runtimeElements) --> 'androidx.annotation:annotation:1.1.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.test:core:1.5.0' (runtime) --> 'androidx.concurrent:concurrent-futures:1.0.0' (runtime) --> 'androidx.annotation:annotation:1.1.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.room:room-testing:2.6.1' (releaseVariantReleaseRuntimePublication) --> 'androidx.room:room-common:2.6.1' (runtimeElements) --> 'androidx.annotation:annotation:1.3.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.room:room-testing:2.6.1' (releaseVariantReleaseRuntimePublication) --> 'androidx.arch.core:core-runtime:2.2.0' (releaseVariantReleaseRuntimePublication) --> 'androidx.annotation:annotation:1.1.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.room:room-testing:2.6.1' (releaseVariantReleaseRuntimePublication) --> 'androidx.sqlite:sqlite:2.4.0' (releaseVariantReleaseRuntimePublication) --> 'androidx.annotation:annotation:1.0.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.room:room-testing:2.6.1' (releaseVariantReleaseRuntimePublication) --> 'androidx.sqlite:sqlite-framework:2.4.0' (releaseVariantReleaseRuntimePublication) --> 'androidx.annotation:annotation:1.2.0'
           Dependency path 'smouldering-durtles:app:unspecified' --> 'androidx.room:room-testing:2.6.1' (releaseVariantReleaseRuntimePublication) --> 'androidx.arch.core:core-runtime:2.2.0' (releaseVariantReleaseRuntimePublication) --> 'androidx.arch.core:core-common:2.2.0' (runtimeElements) --> 'androidx.annotation:annotation:1.1.0'
```

If you had some other way to set up the environment that avoids this issue let me know and I'll clone that to my setup and remove this.